### DISCUSSION TITLE
Engels of nederlands

### DISCUSSION THREAD

**Discussion by Sensinki** [Sensinki/Blok-Tech-Team-4A#37] [Q&A] [answered]
Gaan we alle wiki nederlands of engels schrijven? 
Donna heeft engels geschreven maar Rosalie, nederlands. Ik denk dat het engels beter zou zijn. Wat denken jullie?

**Comment by dinhs-frontend** [comment]
Mij maakt het niet uit

**Comment by D0nnaz** [comment]
We hadden Engels afgesproken in de les

**Reply by Sensinki** [reply]
Ja, zo herinner ik het me ook

**Reply by rosa33783** [reply, marked as answer]
ik ga die van mij aanpassen naar engels!
